packages/amplication-build-manager-service/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@amplication/build-manager-service",
+  "version": "0.14.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json"
+  },
+  "dependencies": {
+    "@amplication/kafka": "^0.14.1",
+    "@aws-sdk/client-codebuild": "^3.100.0",
+    "@aws-sdk/client-s3": "^3.113.0",
+    "@nestjs/common": "8.4.3",
+    "@nestjs/config": "1.1.5",
+    "@nestjs/core": "8.2.3",
+    "@nestjs/microservices": "^8.4.5",
+    "@nestjs/platform-express": "^8.0.0",
+    "jszip": "^3.10.0",
+    "reflect-metadata": "^0.1.13",
+    "rimraf": "^3.0.2"
+  },
+  "devDependencies": {
+    "@nestjs/cli": "^8.0.0",
```

Comment:
lock versions in package json

Reply:
Your call what you want to do with this. I understand the need for consistency and version pinning, on the other hand I understand your argument about manual work because if we pin we might want to pin to the latest minor/patch version, and that needs to be done manually. Time is precious.